### PR TITLE
feat(Bitfield): Improve how BitsBlock works

### PR DESCRIPTION
**Pull request by PalmDevs**

### CODE REVIEW

**Comment by PalmDevs** [comment]
Don't merge yet, I'm fixing something else

**Comment by PalmDevs** [comment]
Okay, tested. It works now. 
@axisiscool @knokbak, please review.

**Review by knokbak — APPROVED** [review]

File: src/classes/blocks/BitsBlock.ts [hunk 1]
```diff
@@ -8,90 +8,133 @@
...
     /**
      * A BitsBlock stores bitfield data provided by Discord.
      * 
-     * @param {...number[]} flags The flags to set.
+     * @param {Object} flags All available flags
+     * @param {Flags | Flags[] | number} bits Bits to add
+     */
+    constructor(flags: BitsBlock<Flags>["ENUM"], bits?: BitResolvable<Flags>) {
+        this.ENUM = flags
+        this.set(bits || 0)
```
Comment (axisiscool) [inline src/classes/blocks/BitsBlock.ts]:
```suggestion
        this.set(bits ?? 0)
```

File: src/util/Constants.ts [hunk 2]
```diff
@@ -380,4 +397,173 @@ export enum WebhookTypes {
...
+    SEND_MESSAGES_IN_THREADS = (1 << 38),
+    /**
+     * Allows for launching activities (applications with the `EMBEDDED` flag) in a voice channel.
+     */
+    START_EMBEDDED_ACTIVITIES = (1 << 39),
+    /**
+     * Allows for timing out users to prevent them from sending or reacting to messages in chat and threads, and from speaking in voice and stage channels.
+     */
+    MODERATE_MEMBERS = (1 << 40),
+}
```
Comment (axisiscool) [inline src/util/Constants.ts]: Can we use bigints for this?

**Review by axisiscool — DISMISSED** [review]

File: src/util/Constants.ts (hunk 2 above)
Comment (PalmDevs) [reply src/util/Constants.ts]: > Can we use bigints for this?

Done in [here](https://github.com/BirbJS/Birb/pull/38/commits/20964149a949f3a0be87238700f799c14a26734e).

**Review by axisiscool — APPROVED** [review]

**Comment by PalmDevs** [comment]
Compile success + check and testing success. Merging...

**Pull request merged**